src/libnvme.map

```diff
@@ -114,6 +114,7 @@
 		nvme_get_log_endurance_group;
 		nvme_get_log_endurance_grp_evt;
 		nvme_get_log_error;
+		nvme_get_log_fid_sup_efcs;
 		nvme_get_log_fw_slot;
```

Comment:
Please make that 'get_log_fid_sup_effects'. 'efcs' is making no sense, and is not something one could figure out as an abbreviation.

Reply:
I was also unhappy with it, especially after I realized that I spelled it NVME_LOG_LID_FID_SUPPORT_EFFECTS for the define.